python/ray/test/test_ray_init.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import pytest
+import redis
+
+import ray
+
+
+@pytest.fixture
+def start_ray_with_password():
+    ray.shutdown()
```

Comment:
Why are we calling `ray.shutdown` at the start of the fixture? We don't do this in any of our other tests.

Reply:
Instead, our fixtures usually do something like

```python
ray.init()

yield None

ray.shutdown()
```